app/experimenter/nimbus-ui/src/components/PageResults/TableVisualizationRow/index.tsx

```diff
@@ -130,8 +130,8 @@ const countField = (
   metricName: string,
   tableLabel: string,
 ) => {
-  const interval = `${Math.round(lower * 1000) / 100} to ${
-    Math.round(upper * 1000) / 100
+  const interval = `${Math.round(lower * 100) / 100} to ${
+    Math.round(upper * 100) / 100
```

Comment:
Do we need `*100` and `/100`? Is this line equal to this?
```
const interval = `${Math.round(lower)} to ${Math.round(upper)}`
```

Reply:
Yes, the reason this is done is to keep 2 decimal places.

Eg. 
```
let a = 1324.324334343;
a = a * 100; // a = 132432.4334343
a = Math.round(a); // a = 132432
a = a / 100; // a = 1324.32
```

Reply:
Ahh makes sense, LGTM then. :)

Reply:
What about `a.toFixed(2)`?